pkg/object/object.go

```diff
@@ -31,6 +33,7 @@ type Container struct {
 type Service struct {
 	Name       string
 	Containers []Container
+	Labels     Label
```

Comment:
in object.go all the types are made of builtin types as opposed to ones like this! I think @tnozicka can shade more light on it!